assemblies/assembly-enabling-authentication.adoc

```diff
@@ -1,39 +1,46 @@
 [id='enabling-authentication']
 = Enabling authentication in {product}
 
-Authentication within {product} facilitates user sign-in, identification, and access to external resources.
-It supports multiple authentication providers.
-
-Authentication providers are typically used in the following ways:
-
-- One provider for sign-in and identification.
-- Additional providers for accessing external resources.
-
-The {product} supports the following authentication providers:
-
-//Auth0:: `auth0`
-//Atlassian:: `atlassian`
-Microsoft Azure::
-`microsoft`
-//Microsoft Azure Easy Auth:: `azure-easyauth`
-//Bitbucket:: `bitbucket`
-//Bitbucket Server:: `bitbucketServer`
-//Cloudflare Access:: `cfaccess`
-GitHub:: `github`
-//GitLab:: `gitlab`
-//Google:: `google`
-//Google IAP:: `gcp-iap`
-Keycloak:: `oidc`
-//Okta:: `okta`
-//OAuth 2 Custom Proxy:: `oauth2Proxy`
-//OneLogin:: `onelogin`
-//SAML:: `saml`
-
-For each provider that you want to use, follow the dedicated procedure to complete the following tasks:
+Your organization certainly requires you to identify and authorize users to give them access to resources, such as {product}.
```

Comment:
```suggestion
You can identify and authorize users to give them access to resources, such as {product}.
```

Reply:
The new sentence removes the fact that this is an hypothesis.

Reply:
Attempting a rewrite that clarifies that we are expressing an assumption:

```suggestion
The chances are high that in your organization, you must identify and authorize users before giving them access to resources, such as {product}.
```